pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: encrypt
-version: 4.1.0
+version: 5.0.0
```

Comment:
This is in accordance to Dart team recommendations: https://dart.dev/null-safety/migration-guide#package-version

Reply:
Ok